include/gz/physics/FixedJoint.hh

```diff
@@ -61,6 +61,45 @@ namespace gz
             const std::string &_name) = 0;
       };
     };
+
+    /////////////////////////////////////////////////
+    /// \brief This feature sets dynamic constraint properties.
+    class GZ_PHYSICS_VISIBLE SetDynamicJointConstraintPropertiesFeature
+        : public virtual FeatureWithRequirements<FixedJointCast>
+    {
+      /// \brief Set dynamic joint constraint properties.
+      public: template <typename PolicyT, typename FeaturesT>
+      class Link : public virtual Feature::Link<PolicyT, FeaturesT>
```

Comment:
from an API perspective, it seems odd that this is a `Link` API and not a `Joint` API. I can see from the implementation that it is more convenient this way, but I think it would be cleaner as a `Joint` method

if it's not currently possible, we should open an issue about that